### ISSUE TITLE
Gradle publication

### ISSUE THREAD

**Issue by vojkny** [jprante/gradle-plugin-jflex#19]
Hi,

as the jcenter/bintray were turned off it seems the plugin cannot be currently used.

Eg.

```
curl -v 'https://plugins.gradle.org/m2/org/xbib/gradle/plugin/gradle-plugin-jflex/1.5.0/gradle-plugin-jflex-1.5.0.module'

< HTTP/1.1 303 See Other
< Location: https://jcenter.bintray.com/org/xbib/gradle/plugin/gradle-plugin-jflex/1.5.0/gradle-plugin-jflex-1.5.0.module
```

An the jcenter link is then dead.

**Comment by vojkny** [comment]
🙏

**Comment by ghulands** [comment]
@jprante Are you able to push out 1.5.0 please?

**Comment by jprante** [comment]
I just published 1.6.0

https://plugins.gradle.org/plugin/org.xbib.gradle.plugin.jflex
